README.md

```diff
@@ -49,19 +49,22 @@ client = Kubeclient::Client.new uri
 It is also possible to use https and configure ssl with:
 
 ```ruby
-client = Kubeclient::Client.new 'https://localhost:8443/api/' , "v1beta3"
-client.ssl_options(
+ssl_options = {
```

Comment:
Can you make this consistent with `auth_options` below? (either both defined in `ssl_options` and `auth_options` or just passed to the constructor)